src-tauri/tauri.conf.json

```diff
@@ -17,6 +17,11 @@
                 "execute": true,
                 "sidecar": true,
                 "scope": [
+                    {
+                        "name": "nmap",
```

Comment:
I am a bit confused about the changes to tauri.conf.json. This should be sorted in alphabetical order so can you please remove nmap into the correct position?